src/main/java/core/basesyntax/dao/DAoService.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.models.Fruit;
+import core.basesyntax.models.FruitRecord;
+import java.util.Set;
+
+public interface DAoService {
```

Comment:
What is the type of this DAO? Correct DAO names: `FruitDao`, `RecordDao`, etc.
Standard DAO methods is: `get()`, `getAll()`, `save()`/`put()`